server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -235,6 +236,7 @@
   public void backfillExpirationTest() throws Exception {
     byte[] certs = Resources.toByteArray(Resources.getResource("fixtures/expiring-certificates.crt"));
     byte[] pubring = Resources.toByteArray(Resources.getResource("fixtures/expiring-pubring.gpg"));
+    byte[] p12 = Resources.toByteArray(Resources.getResource("fixtures/expiring-keystore.p12"));
```

Comment:
I added JCEKS, but I don't have a JKS example handy right now... but since it hits the same code path anyway, do we really need an example of every file?